internal/venice-common/src/main/resources/avro/RecordChangeEvent.avsc

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "RecordChangeEvent",
+  "doc": "Schema which describes a record change event in a Venice store.",
+  "namespace": "com.linkedin.venice.client.change.capture.protocol",
+  "type": "record",
+  "fields":
+  [
+    {
+      "name": "previousValue",
+      "doc": "",
+      "default": null,
+      "type": [
+        "null",
+        {
+          "name": "ValueBytes",
+          "doc": "Optional field which describes the previous state of a given row.  Can be null if before image isn't enabled, or if the record was null to begin with.",
+          "type": "record",
+          "fields": [
+            {
+              "name": "value",
+              "type": "bytes"
+            },
+            {
+              "name": "schemaId",
```

Comment:
Just curious why we only have `schemaId` in `previousValue`, is it for store value schema update, so that currentValue can just use current valueSchema?

Reply:
So a client using change capture will specify a type and we'll need to deserialize all events from whatever is in the change log to whatever is understood by the client.  So for a ser/de path we need both the writing schema (whatever was outputted) and the reader schema.  So having the path on both before/after is a good way to make sure we never hiccup.

Reply:
I understand we need R/W schema, my confusion is we have `schemaId` in `previousValue`, do we need `schemaId` in `currentValue`?

Reply:
Oh good point.  We probably should have it for both.

Reply:
Sorry typed too fast.  We do need it for both.  ValueSchemaBytes is a type which will include for both.  I guess if we were particular we could figure out currentValueSchema just by looking at the current version.... But...  I think it might save us some trouble if it's explicit?

Reply:
I think make it explicit might be more robust if consumer read very old events.

Reply:
{
      "name": "currentValue",
      "doc": "The current value of this record as of this change event.  Can be null to signify a delete.",
      "type": ["null","ValueBytes"]
    },

the type is ValueBytes which is: 

          "fields": [
            {
              "name": "value",
              "type": "bytes"
            },
            {
              "name": "schemaId",

CurrentValue and previousValue are the same type.  They both have schemaId